packages/ckeditor5-dev-env/lib/release-tools/utils/versions.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+const changelogUtils = require( './changelog' );
+const getPackageJson = require( './getpackagejson' );
+
+const versions = {
+	/**
+	 * Returns a last created version in changelog file.
+	 *
+	 * @returns {String|null}
+	 */
+	getLastFromChangelog() {
+		const changelog = changelogUtils.getChangelog();
+		// TODO: Support for versions: alpha/beta/rc.
+		const regexp = /## \[?(\d+\.\d+\.\d+)\]?/;
+
+		const matches = changelog.match( regexp );
+
+		return ( matches ) ? matches[ 1 ] : null;
+	},
+
+	/**
+	 * Returns a name of the last created tag.
+	 *
+	 * @returns {String|null}
+	 */
+	getLastFromTag() {
```

Comment:
This name makes no sense. If you have a problem with naming a function, use the docs as an inspiration.